languages/ocaml.scm

```diff
@@ -433,6 +433,10 @@
   "and" @prepend_spaced_softline
 )
 
+(type_definition
+  "and" @prepend_spaced_softline
```

Comment:
Can't we simply always prepend a softline before `and`?

Reply:
I think we can. Let me have a check in the grammar of Ocaml before doing this modification.

Reply:
A search for `and` in https://github.com/tree-sitter/tree-sitter-ocaml/blob/master/ocaml/grammar.js tells me that and can appear in mutually recursive `value_definition`, `type_definition`, `module_definition`, `module_type_definition` and `class_definition`. I thunk we want a soft break in all those cases.